lib/levels.js

```diff
@@ -49,13 +49,14 @@ const initialLsCache = Object.keys(nums).reduce((o, k) => {
 }, {})
 
 function genLsCache (instance) {
-  const levelName = instance[changeLevelNameSym]
-  instance[lsCacheSym] = Object.keys(instance.levels.labels).reduce((o, k) => {
-    o[k] = instance[useLevelLabelsSym]
-      ? `{"${levelName}":"${instance.levels.labels[k]}"`
-      : flatstr(`{"${levelName}":` + Number(k))
-    return o
-  }, Object.assign({}, instance[lsCacheSym]))
+  const serializer = instance[serializersSym][levelSerializerSym]
+  const { labels } = instance.levels
+  const cache = {}
+  Object.keys(labels).forEach(number => {
+    const level = serializer(labels[number], Number(number))
+    cache[number] = JSON.stringify(level).slice(0, -1)
```

Comment:
there's a much faster escaping function here https://github.com/pinojs/pino/blob/master/lib/tools.js#L48

use that instead of JSON.stringify

Reply:
I can't as the user here is retuning a js object, `asString` is built for escaping strings, isn't it?

Reply:
oh sorry I thought level was a number or string

if it's an object why not just access the property from the object?